index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<link rel="stylesheet" href="styles.css"></link>
```

Comment:
I can't see any "styles.css" file in your core path. Kindly ask for some styles for your web page for a better looking.